public/components/layout.js

```diff
@@ -44,6 +46,26 @@ class GeoHeader extends HTMLElement {
 
 customElements.define("geo-header",GeoHeader)
 
+class GeoNav extends HTMLElement {
+    #navTmpl = `
+    <link rel="stylesheet" href="https://unpkg.com/chota@latest">
```

Comment:
I noticed we did this in some templates.  I wonder if we actually need to.  Shower thoughts for another day.